### DISCUSSION TITLE
Add a custom platform?

### DISCUSSION THREAD

**Discussion by WKampel** [jaredh159/tailwind-react-native-classnames#329] [Announcements]
The web portion of my app has an emulator view (just a phone that content can be passed to).

The ios / android prefixes obviously don't work since it's web. md/sm prefixes also don't work since it's checking the browser dimensions not the emulator's dimensions. 

Is there a way I could add 2 custom prefixes?

**preview** - This will be active if within a PhoneProvider.
**mobile** - This will be ios, android or preview.

**Comment by jaredh159** [comment]
I'm honestly not entirely sure, without digging into the codebase a bit and reminding myself how certain things work that I haven't touched in a while. My hunch is that this would not be possible without changes to the library itself (as opposed to some configuration), but I'm not totally sure. Tailwind has a way to define custom prefixes, but that doesn't mean it would transfer over to this library without some manual reimplementation, unless we just sort of got lucky and something already implemented happens to handle it.

**Comment by samuelpetroline** [comment]
I would be in favor of this. I currently have a RN app that ports to web and would love to use Platform-specific breakpoints for web e.g `web:w-full` and `mobile:w-1/2` etc.

**Reply by WKampel** [reply]
The web prefix is already supported.
